app/channels/application_cable/channel.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
You can add these lines in rubocop.yml to not include them in all files this useless line
```yml
Style/FrozenStringLiteralComment:
  Enabled: false
```